client/src/components/user/profile/settings/SettingsRadioGroup.tsx

```diff
@@ -0,0 +1,84 @@
+import { useTheme } from "@emotion/react";
+import {
+	Radio,
+	type RadioCardProps,
+	RadioIndicator,
+	Tooltip,
+} from "@mantine/core";
+import type { LucideIcon } from "lucide-react";
+import type { RadioGroupOption } from "@/components/Today/BlockHeader";
+import type { HabitFilter } from "@/components/Today/habits/habit-filter";
+import type { TaskFilter } from "@/components/Today/tasks/task-filter";
+import type { MainTheme } from "@/lib/style/theme";
+import { colors } from "@/lib/theme/colors";
+import Containers from "@/lib/theme/components/container.style";
+
+function RadioOption({
+	Icon,
+	tooltipLabel,
+	isChecked,
+	...props
```

Comment:
Like in the other comment, suggestion: rename this to `radioCardProps` to make it clear why we're destructuring like this.